kv-components/vue/KvLoadingSpinner.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+	<svg
+		class="animate-spin"
+		:class="`${cssClassSize} ${cssClassColor}`"
```

Comment:
I don't think this will get picked up by purgeCss. https://tailwindcss.com/docs/optimizing-for-production#writing-purgeable-html

I believe purge only checks the `<template>` area for css class names.

KvButton shows a way to do this all in the template.

Reply:
It'd be good to verify that though. We'd need to set up purge on a different repo, import this component, and run a prod build, then look at the output CSS file.